contracts/transfer/src/tests.rs

```diff
@@ -284,20 +288,22 @@ fn withdraw_from_transparent_to_contract() {
     // https://github.com/dusk-network/rusk/issues/194
     let spend_proof_execute = vec![0xfa];
     let spend_proof_stct = vec![0xfa];
-    let call = TransferExecute {
-        anchor: root,
-        nullifiers: [genesis_nullifier].into(),
-        fee: bob_fee,
-        crossover: Some(bob_crossover),
-        notes: [genesis_unspent_note].into(),
-        spend_proof: spend_proof_execute,
-        call: Some(Call::SendToContractTransparent {
-            address: bob_address,
-            value: bob_value,
-            pk: bob_psk.A().into(),
-            spend_proof: spend_proof_stct,
-        }),
-    };
+    let call = Call::send_to_contract_transparent::<MemStore>(
+        bob_address,
+        bob_value,
+        bob_psk.A().into(),
+        spend_proof_stct,
+    )
+    .unwrap();
```

Comment:
Use expect. In case we can capture the `panic` appended &str it could be used as err message.

Reply:
Cool, I'll add these expect and CHANGELOG on the next PR related to the integrated tests. Ok?

Reply:
Perfect!